CMakeLists.txt

```diff
@@ -19,13 +19,21 @@ if(NOT WIN32)
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+option(DISABLE_SANITIZERS "disables the use of gcc sanitizers" OFF)
+# Only enable sanitizers on x64 architectures
+# https://github.com/google/sanitizers/issues/794
+if(NOT ${CMAKE_SIZEOF_VOID_P} EQUAL 8)
+  set(DISABLE_SANITIZERS ON)
+endif()
+
```

Comment:
Imo this new option is completely orthogonal to the feature implemented in this PR. Therefore it should be moved into a separate PR. Maybe it would make sense to add this option to `ament_cmake_ros` instead?

Reply:
Removed sanitizers https://github.com/ros2/rcpputils/pull/49/commits/2d5338096c60621b95e6e71d38cbec5e5fd1d5d6